lib/data/account_http_helper.dart

```diff
@@ -0,0 +1,58 @@
+import 'dart:async';
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:http/http.dart' as http;
+import 'package:learning_analytics/data/account.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+
+class AccountHttpHelper {
+  final String authority = 'api.learning-analytics.gang-of-fork.de';
+
+  Future<bool> postAccount(RegisterAccount account) async {
+    String newPath = '/auth/register/63496390b3e7ebc4e3d8cb50';
+    Uri uri = Uri.https(authority, newPath);
+    var email = account.email;
+    var password = account.password;
```

Comment:
nutzt du die noch irgendwo?

Reply:
ja in 29 wird damit der Account erzeugt

Reply:
Okay gecheckt. Müsste aber auch als Antwort zurück kommen